### DISCUSSION TITLE
Recommended way to clear the working directory after run on persistent instance?

### DISCUSSION THREAD

**Discussion by maflcko** [cirruslabs/cirrus-ci-docs#822] [Q&A] [answered]
When using a persistent instance, the default working dir is `/tmp/cirrus-ci-build/`, if it exists a random new dir of the form `/tmp/cirrus-task-nnnnnnnnnnn/` is used. This has two downsides:

* The disk usage on the persistent instance (https://cirrus-ci.org/guide/persistent-workers/#persistent-workers) increases until no space is available
* Caches residing inside the working dir are invalidated on each run

**Comment by fkorotkov** [comment]
Oh! That looks like a bug which has roots in the initial design that all the instances are ephemeral. Fixing it in https://github.com/cirruslabs/cirrus-ci-agent/pull/112

**Comment by fkorotkov** [comment, marked as answer]
Version [0.43.0](https://github.com/cirruslabs/cirrus-cli/releases/tag/v0.43.0) should have it fixed once it's built.

**Reply by ghost** [reply]
> Version [0.43.0](https://github.com/cirruslabs/cirrus-cli/releases/tag/v0.43.0) should have it fixed once it's built.
